packages/isomorphic/src/utils/browser.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  bytesToHex as nobleBytesToHex,
+  hexToBytes as nobleHexToBytes,
+  randomBytes as nobleRandomBytes,
+} from '@noble/hashes/utils'
+import { BytesToHexFn, HexToBytesFn, RandomBytesFn } from './types'
+
+// eslint-disable-next-line func-style
```

Comment:
Why does this need to be disabled?

Reply:
I had to write it in functional style so that I could type the function and force it to adhere to the signature in both implementations.  eslint didnt like this but gave me no alternative to type a method.  Originally I disabled this rule for isomorphic.  When the non treeshaking code was reverted I think that was lost.